core/src/test/java/org/jobrunr/scheduling/interval/IntervalTest.java

```diff
@@ -0,0 +1,105 @@
+package org.jobrunr.scheduling.interval;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.stream.Stream;
+
+import static java.time.ZoneOffset.UTC;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+class IntervalTest {
+
+    private static final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+    private static final String TEN_SECONDS = "PT1S";
```

Comment:
Should be PT10S.

If I change this to PT10S, no tests fail which I would expect them to do. 

Can you also have an expectedResult in the startInstantDurationAndResultInstant and compare the expected value to the actual value? That it is only after the current instant will not prevent me from making mistakes in the future if something changes.

Reply:
Since `Instant.now()` is used here, the only way to get the expected result would be to calculate it using the same logic that is supposed to be verified by this test case. I'm not entirely sure if that would be a good idea. 

I added a new test instead that uses Instant.now() as base date and checks if the next instant is ten seconds later.

Let me know what you think :)